backend/src/main/kotlin/rocks/didit/sefilm/domain/dto/AttendInfoDTO.kt

```diff
@@ -0,0 +1,5 @@
+package rocks.didit.sefilm.domain.dto;
+
+import rocks.didit.sefilm.domain.Foretagsbiljett
+
+data class AttendInfoDTO(val paymentOption: String)
```

Comment:
paymentOption is a bit vague what it means. Consider changing the `String` to an `Enum` instead. This has the side effect of making your `when`s exhaustive (you don't need an `else`-branch)

Example:
```kotlin
enum class PaymentOption {
    Swish, Företagsbiljett, Other
}
```